thespian/system/multiprocCommon.py

```diff
@@ -512,6 +510,69 @@ def shutdown_signal_detected(signum, frame):
     return shutdown_signal_detected
 
 
+def get_min_log_level(logDefs):
+    """
+    Determine the minimum logging level based on the rules in:
+    https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig
+
+    Note: The level and handlers entries are interpreted as for the root logger,
+    except that if a non-root logger’s level is specified as NOTSET, the system
+    consults loggers higher up the hierarchy to determine the effective
+    level of the logger.
+
+    :param logDefs: a logging configuration dictionary object
+    :return: integer value of the lowest log level, or the default level: logging.WARNING.
+
+    >>> import logging
+    >>> logDefs = {'version': 1, 'handlers': {'h1': {'class':
+    ... 'logging.FileHandler', 'filename': 'example.log', 'formatter':
+    ... 'normal', 'filters': ['notActorLog'], 'level': "INFO"}, 'h2': {'class':
+    ... 'logging.FileHandler', 'filename': 'example.log', 'formatter':
+    ... 'actor', 'filters': ['isActorLog'], 'level': "INFO"}},
+    ... 'loggers': {'': {'handlers': ['h1', 'h2'], 'level': "DEBUG"}}}
```

Comment:
This is better, but still not very readable.  I was thinking that a more classical Python formatting would be more understandable.

Also a blank line between the test cases might also help readability.

Reply:
Sure.